tests/e2e/framework/kubernetes.go

```diff
@@ -111,6 +113,7 @@ var (
 	imagePullPolicy     = flag.String("image_pull_policy", "", "Specifies an override for the Docker image pull policy to be used")
 	multiClusterDir     = flag.String("cluster_registry_dir", "",
 		"Directory name for the cluster registry config. When provided a multicluster test to be run across two clusters.")
+	splitHorizon             = flag.Bool("split_horizon", false, "Set up a split horizon EDS multi-cluster test environment")
```

Comment:
The README in the tests directory lists most e2e test options.   Its good practice to add your new variable there as well.

Reply:
Addressed in https://github.com/istio/istio/pull/12709/commits/99f7febcdaf5c24a7ff13ec5825b2ee83fe0dcd1.